ethportal-peertest/src/jsonrpc.rs

```diff
@@ -364,6 +370,15 @@ fn validate_portal_routing_table_info(result: &Value, _peertest: &Peertest) {
     assert!(result.get("numConnected").unwrap().is_u64());
 }
 
+pub fn validate_portal_offer(result: &Value, _peertest: &Peertest) {
+    println!("result: {}", result);
```

Comment:
Maybe left in by accident, and should be deleted? If you think we should keep it, then I might expect it to be a `debug` log, with a bit more info.